README.md

```diff
@@ -0,0 +1,580 @@
+lein new xiana app
+```
+
+### As dependency
+
+Add it to your project as dependency from clojars:
+
+[![Clojars Project](https://img.shields.io/clojars/v/com.flexiana/framework.svg)](https://clojars.org/com.flexiana/framework)
+
+### Examples
+
+Visit [examples folder](https://github.com/Flexiana/framework/tree/main/examples), and see how you can do
+
+- [Access and data ownership control](examples/acl/README.md)
+- [Coercion, validation](examples/controllers/README.md)
+- [Session handling with varying interceptors](examples/sessions/README.md)
+
+## Contribution
+
+### Development dependencies
+
+#### Mandatory
+
+- Clojure 1.10
+- Postgresql >= 11.5
+- leiningen >= 2.9.0
+
+#### Optional
+
+- Docker >= 19.03.11
+- Docker-compose >= 1.21.0
+
+#### Libraries
+
+##### Mandatory
+
+| Name                            | Version | Related    |
+|---------------------------------|---------|------------|
+| funcool/cats                    |   2.4.1 | Monad      |
+| funcool/cuerdas                 | RELEASE | Monad      |
+| metosin/reitit                  |  0.5.12 | Routes     |
+| potemkin/potemkin               |   0.4.5 | Helper     |
+| com.draines/postal              |   2.0.4 | Email      |
+| duct/server.http.jetty          |   0.2.1 | WebServer  |
+| seancorfield/next.jdbc          | 1.1.613 | WebServer  |
+| honeysql/honeysql               | 1.0.444 | PostGreSQL |
+| nilenso/honeysql-postgres       |   0.2.6 | PostGreSQL |
+| org.postgresql/postgresql       |  42.2.2 | PostGreSQL |
+| crypto-password/crypto-password |   0.2.1 | Security   |
+
+##### Optional
+
+| Name                | Version | Provide |
+|---------------------|---------|---------|
+| clj-kondo/clj-kondo | RELEASE | Tests   |
+
+### Setup
+
+```shell
+$ git clone git@github.com:Flexiana/framework.git; cd framework
+$ ./script/auto.sh -y all
+```
+
+The first command will clone `Flexiana/framework` repository and jump to its directory. The second command
+calls `auto.sh` script to perform the following sequence of steps:
+
+1. Download the necessary docker images
+2. Instantiate the database container
+3. Import the initial SQL schema: `./docker/sql-scripts/init.sql`
+4. Populate the new schema with 'fake' data from: `./docker/sql-scripts/test.sql`
+5. Call `lein test` that will download the necessary *Clojure*
+   dependencies and executes unitary tests.
+
+See `./script/auto.sh help` for more advanced options.
+
+Remember it's necessary to have `docker/docker-compose` installed in your host machine. Docker daemon should be
+initialized a priori, otherwise the chain of commands fails.
+
+It should also be noted that after the first installation everything will be cached preventing unnecessary rework, it's
+possible to run only the tests, if your development environment is already up, increasing the overall productivity.
+
+```shell
+./script/auto.sh -y tests
+```
+
+### Deps
+
+We define some aliases to make possible to use `deps.edn` directly
+(recommend).
+
+### leiningen
```

Comment:
Is this aimed at a user trying to create their own project making use of the framework? Might be more use in the developer guide. (or maybe both)